multidict/_pair_list.c

```diff
@@ -61,6 +67,66 @@ str_cmp(PyObject *s1, PyObject *s2)
 }
 
 
+static INLINE PyObject *
+key_to_str(PyObject *key)
+{
+    PyObject *type = Py_TYPE(key);
+    if (PyUnicode_CheckExact(key)) {
+        Py_INCREF(key);
+        return key;
+    }
+    if (type == _istr_type) {
+        return PyObject_Str(key);
+    }
+    if (PyUnicode_Check(key)) {
+        return PyObject_Str(key);
+    }
+    PyErr_SetString(PyExc_TypeError,
+                    "MultiDict keys should be either str "
+                    "or subclasses of str");
+    return NULL;
+}
+
+
+static PyObject *
+key_to_identity(PyObject *key)
```

Comment:
I cant see a difference between this function and the previous key_to_str. Do we have to merge them in one?

Reply:
Done